packages/verification-app/src/assets/scss/app.scss

```diff
@@ -2,3 +2,11 @@
 @import "~bootstrap/scss/bootstrap-reboot";
 // Components
 @import "components/index";
+
+html {
+    height: 100%;
+
+    body, .app {
+        height: 100%;
+    }
+}
```

Comment:
```suggestion
html,
body,
.verification-app {
    min-height: 100%;
}
```

Reply:
min-height didn't work, changed it to height